### PR TITLE
New articles added

### PR DESCRIPTION
**Pull request by Mike4o**
- How to Install the Telerik Application Enrollment Token
- Cannot Install Company App on Windows Phone 8
- Cannot Run Installed Apps on Windows Phone 8

### CODE REVIEW

File: how-to/how-to-wp-install-aet.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    <td>Hybrid (Cordova app)</td>	
+  </tr>
+  <tr>
+    <td>Mobile OS</td>
+    <td>Windows Phone</td>	
+  </tr> 
+</table>
+
+## Description
+To deploy Windows Phone 8 application packages via QR code, you need to have the Telerik Application Enrollment Token (AET) installed on the device. It configures the Telerik AD workplace account, which is essential for installing and running AppBuilder apps on test devices.
```
Comment (AntonDobrev) [inline how-to/how-to-wp-install-aet.md]: Perhaps this should be added as a variable in the YML file.

**Comment by nadyaA** [comment]
:+1:  great work

File: how-to/how-to-wp-install-aet.md (hunk 1 above)
Comment (AntonDobrev) [inline how-to/how-to-wp-install-aet.md]: I think that "Windows Phone 8 apps built with {{site.tp}}"" or similar sounds better. Anyways, please use the variable for the respective product or service.

**Comment by AntonDobrev** [comment]
:+1:

**Pull request merged**